fuel-bft/Cargo.toml

```diff
@@ -11,6 +11,17 @@ repository = "https://github.com/FuelLabs/fuel-bft"
 description = "A Rust implementation of the Tendermint consensus protocol."
 
 [dependencies]
+async-trait = "0.1"
+fuel-crypto = { version = "0.4", default-features = false }
+fuel-types = { version = "0.3", default-features = false }
+tracing = { version = "0.1", default-features = false, optional = true }
 
 [dev-dependencies]
-fuel-bft-mock = { path = "tests/mock" }
+fuel-bft-test-structures = { path = "../fuel-bft-test-structures" }
```

Comment:
These are `dev-dependencies` and shouldn't interact with the publishing process. Anyway, I think its worthy to move the tests to `fuel-bft-test-structures` so we also unblock the `no-std` checks for this crate

Reply:
Restructured the test deps to live in `fuel-bft-test-structures` in case they depend on that:

https://github.com/FuelLabs/fuel-bft/pull/9/commits/69a644e1d4e216e10f1dc2580db6ccae3e54b043